src/common/utils.ts

```diff
@@ -85,3 +85,7 @@ export const parseDate = (date: string | Date): Date | undefined => {
 
   return parsedDate;
 };
+
+export const toGQLEnum = (value: Record<string, string>, name: string) => {
+  return `enum ${name} { ${Object.keys(value).join(' ')} }`;
+};
```

Comment:
Take typescript enum and convert to gql enum template which can be added to schema.